apps/manuscripts/search_indexes.py

```diff
@@ -6,13 +6,18 @@
 
 
 class ItemPartIndex(indexes.ModelSearchIndex, indexes.Indexable):
+    django_ct = indexes.CharField()
+    django_id = indexes.CharField()
     text = indexes.CharField(document=True, use_template=False)
     id = indexes.IntegerField(model_attr="id")
     repository_name = indexes.CharField(model_attr="current_item__repository__name", faceted=True)
     repository_city = indexes.CharField(model_attr="current_item__repository__place", faceted=True)
     shelfmark = indexes.CharField(model_attr="current_item__shelfmark")
     catalogue_numbers = indexes.CharField(model_attr="historical_item", faceted=True)
     date = indexes.CharField(model_attr="historical_item__date")
```

Comment:
This line probably needs to be changed as well
```suggestion
    date = indexes.CharField(model_attr="historical_item__date__date")
```

Reply:
Sure, I was wondering about that as well. It works all the same because we have `return self.date` in the common, but with direct access to the `date` field is more precise.